### PR TITLE
feat: add mfa support

### PR DESCRIPTION
**Pull request by EHadoux**

### CODE REVIEW

File: tests/test_scribeauth.py [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
 
+import pyotp
 from botocore.awsrequest import AWSRequest
 from dotenv import load_dotenv
+
 from scribeauth import (
+    MissingIdException,
+    ResourceNotFoundException,
     ScribeAuth,
     Tokens,
```
Comment (ailinvenerus) [inline tests/test_scribeauth.py:13]: _Tokens_ is imported but we're not using this line anymore in the tests

**Review by ailinvenerus — COMMENTED** [review]
Can we use _load_dotenv(override=True)_ as in ScribeMi to override the env var in the tests?

File: tests/test_scribeauth.py (hunk 1 above)
Comment (EHadoux) [reply tests/test_scribeauth.py:13]: they probably will when you fix the types

**Review by alexbostock — DISMISSED** [review]
Agreed with Ailin's comments; nothing else to add

**Pull request merged**